contracts/bonding/BondingManager.sol

```diff
@@ -820,27 +820,29 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     function updateDelegatorWithTokenPoolsShares(address _delegator, uint256 _endRound) internal {
         Delegator storage del = delegators[_delegator];
 
-        uint256 currentBondedAmount = del.bondedAmount;
-        uint256 currentUnbondedAmount = del.unbondedAmount;
+        if (del.lastClaimTokenPoolsSharesRound > 0) {
```

Comment:
If this condition is not true, then the delegator has never been bonded to anyone before